lesson5/hometask_oefimenko/person.rb

```diff
@@ -0,0 +1,83 @@
+require 'yaml'
+
+module Person
+
+  PATH_TO_SERIALIZED = 'serialized'
+
+  module Factory
+
+    class PersonBase
+      attr_writer :fullname, :sex, :age
+
+      def initialize(label, block)
+        @label = label
+        self.instance_eval &block
+      end
+
+      def fullname(name = '')
```

Comment:
I agree, I think it's terrible idea :) It was quick'n'dirty solution considering requested DSL.
But also I didn't want to create additional object, which would be just a proxy between #sex, #age and #sex=, #age= methods. 
But probably there are not so many options. I'll think on it a little bit.